application/common/components/adapters/WorkdayIdStore.php

```diff
@@ -121,6 +121,14 @@ public function getAllActiveUsers()
                 var_export($allActiveUsers, true)
             ), 1532982679);
         }
+
+        foreach ($allActiveUsers as $key => $user) {
+            $allActiveUsers[$key]['Groups'] = implode(",", array_merge(
+                explode(" ", $user["reap_CompanyId"] ?? ""),
+                explode(" ", $user["reap_tree"] ?? "")
```

Comment:
What do `reap_CompanyId` and `reap_tree` refer to? If possible, it seems worth getting those fields renamed in the Workday API to more meaningfully reflect their purpose.

Reply:
Good question. They're cryptic to me also. @fillup do you think it's possible to have them renamed? and possibly even combine them?

Reply:
well, they are both specifically for REAP, so the name is accurate there, although if another app wanted to use them someday they'd be awkward, but I'll ask and see if they can change them I guess.